### DISCUSSION TITLE
Will CSD have any events for AWS re:Invent?

### DISCUSSION THREAD

**Discussion by fltzr** [cloudscape-design/components#1753] [General]
Hello!!

I am currently attending AWS re:Invent this year. One of the demos from the AI/ML team was using CSD to showcase their demo and thought it was extremely cool!

Made me curious to see if there will be any events this year for CSD specifically or anyone from the team that will be here to connect with? 

Thanks so much!

**Comment by YueyingLu** [comment]
Hey @fltzr,
thank you for reaching out! We are glad you like the demo with Cloudscape.
We would like to connect. Please feel free set up some time via PeerTalk with Olga Madejska.

**Reply by fltzr** [reply]
Will do, thank you!
